src/lib/viewers/controls/page/PageControls.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import Browser from '../../../Browser';
+import { BROWSERS } from '../../../constants';
+import { decodeKeydown } from '../../../util';
+import fullscreen from '../../../Fullscreen';
+import IconArrowDown24 from '../icons/IconArrowDown24';
+import IconArrowUp24 from '../icons/IconArrowUp24';
+import './PageControls.scss';
+
+export type Props = {
+    onPageChange: (arg0: number) => void;
+    pageCount: number;
+    pageNumber: number;
+    viewer: HTMLElement;
+};
+
+export default function PageControls({ onPageChange, pageCount, pageNumber, viewer }: Props): JSX.Element {
+    const isSafariFullscreen = Browser.getName() === 'Safari' && fullscreen.isFullscreen();
```

Comment:
Is this still needed in the latest version of Safari?

Reply:
Just tested and I'm not sure what this was supposed to fix, but it seems it's no longer needed, so removed!